src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -37,16 +37,14 @@
 import org.springframework.http.MediaType;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
 
 @Tag("unit")
 @ActiveProfiles({"test", "unit"})
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = Main.class)
-@SpringBootTest
+@SpringBootTest(classes = Main.class, properties = "terra.common.prometheus.endpointEnabled=false")
```

Comment:
Is there a reason that this property override isn't instead put in a test-specific property file?

Reply:
No reason, good idea -- there is an `application-test.yml` file in Janitor.